cli/src/deploy/mod.rs

```diff
@@ -29,22 +29,22 @@ const UNAUTHORIZED_OPERATION: &str =
 
 #[allow(non_snake_case)]
 #[derive(Deserialize, Debug, Clone, Serialize)]
-pub struct AWSCredentials {
+pub struct AWSCredential {
```

Comment:
So "credentials" is a weird word. https://www.merriam-webster.com/dictionary/credential If you look at the second definition there with `credential` as a noun, the *plural* definition is what applies here, not the singular one.

You basically never hear about a singular "credential" it's always plural in English for weird reasons. You can see evidence of that in the AWS documentation: https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html they only use the plural form of the word even when used for a singular validation.

I *think* this is because computer credentials always require at least two parts, but I'm not sure why that's how it is. Long story short, please revert *all* of these changes with the de-pluralization of credentials.

Reply:
Interesting